### PR TITLE
Cleaned up icon and display name.

### PR DESCRIPTION
**Pull request by Slulego**
Beautified the icon.
Capitalized the display name.

### CODE REVIEW

**Comment by HookyQR** [comment]
I like it. Will package and push out some time today.

File: icon.svg [hunk 1]
```diff
@@ -1,35 +1,51 @@
...
+	<path class="st1" d="M100.9,49.8l-10,3.1L75.4,31.4L56.8,56.6l-31.1,7.9l14.9,24.3l-4.4,6.8l19.3,31.9
+		c31.2,4.3,60.8-14.8,69.9-44.9L100.9,49.8z"/>
+	<path class="st2" d="M64,0C28.7,0,0,28.7,0,64s28.7,64,64,64s64-28.7,64-64S99.3,0,64,0z M64,122.5C31.7,122.5,5.5,96.3,5.5,64
+		S31.7,5.5,64,5.5s58.5,26.2,58.5,58.5S96.3,122.5,64,122.5z M54.8,127.4L54.8,127.4"/>
+	<path class="st3" d="M125.2,82.6l-4.2-5.7c-7.1,31.4-38.2,51.1-69.6,44.2l3.8,6.3C86.4,131.7,116.1,112.8,125.2,82.6L125.2,82.6z"
+		/>
+</g>
+<g id="Flower">
+	<g id="pet">
+		<path class="st3" d="M64,66.4c0,0-13.7-13.6-13.7-27.3c0-8.2,5.5-13.6,13.7-13.6S77.7,31,77.7,39.2C77.7,52.8,64,66.4,64,66.4"/>
```
Comment (HookyQR) [inline icon.svg]: All of the petal elements are south of center,center by 2.4px. Is this intentional? It is noticeable.

Comment (Slulego) [inline icon.svg]: My goodness how did I not notice that? Does it look right now?
Commit: 48955f84a6ee28ac4d47cec5c3340c76544ec0a1

File: icon.svg [hunk 2]
```diff
@@ -1,35 +1,57 @@
...
+		S31.7,5.5,64,5.5s58.5,26.2,58.5,58.5S96.3,122.5,64,122.5z M54.8,127.4L54.8,127.4"/>
+	<path class="st2" d="M125.7,80.9L101.3,48l-10,3.1L75.8,29.7L57.2,54.8l-31.1,7.9L41,87l-4.4,6.8l20.6,33.8
+		c2.3,0.2,4.5,0.4,6.9,0.4C93.5,128,118.3,108,125.7,80.9z"/>
+	<path class="st3" d="M125.7,80.9l-4.3-5.8c-5.2,27-28.9,47.4-57.4,47.4c-3.6,0-7.1-0.3-10.6-1l3.7,6.1c2.3,0.2,4.5,0.4,6.9,0.4
+		C93.5,128,118.3,108,125.7,80.9z"/>
+</g>
+<g id="Flower">
+	<g id="Pedals">
+		<g id="pet">
+			<path class="st3" d="M64,64.1c0,0-13.7-13.6-13.7-27.3c0-8.2,5.5-13.6,13.7-13.6s13.7,5.5,13.7,13.7C77.7,50.5,64,64.1,64,64.1"
```
Comment (HookyQR) [inline icon.svg]: Each of the petals is still of by 0.1px. (64.1 vs 64) The centre circle is now correct though.

Comment (Slulego) [inline icon.svg]: Sorry, I'm traveling so I was sort of rushing through it. ...working on a fix.

**Pull request merged**